m2c/asm_file.py

```diff
@@ -548,6 +554,12 @@ def process_label(label: str, *, kind: LabelKind) -> None:
                         data = parse_incbin(args, options, warnings)
                         if data is not None:
                             asm_file.new_data_bytes(data)
+                    elif directive == ".obj":  # decomp-toolkit label format
+                        if len(args) == 2:
```

Comment:
`assert len(args) == 2` instead to have some minimum of error handling. Or `raise DecompFailure("...")` in an else branch but that feels like effort.

Reply:
Okay, I will use assert